static/js/containers/FinancialAidCalculator.js

```diff
@@ -109,6 +124,16 @@ const validationMessage = (key, errors) => {
   </div>;
 };
 
+const apiError = ({ message, code }: FetchError) => (
+  <div className="api-error">
```

Comment:
Can we use the [`aria-errormessage`](https://www.w3.org/TR/wai-aria-1.1/#aria-errormessage) attribute on the text field to point to this element, when there's an error? That way, screen readers (and other assistive tech) can inform the user _why_ the text field is in an invalid state.